ceci/stage.py

```diff
@@ -51,8 +52,16 @@ def __init__(self, args):
         # command line arguments and optional 'config' file
         self._configs = self.read_config(args)
 
-        if args.get('mpi', False):
-            import mpi4py.MPI
+        use_mpi = args.get('mpi', False)
+        if use_mpi:
+            try:
+                # This isn't a ceci dependency, so give a sensible error message if not installed.
+                import mpi4py.MPI
```

Comment:
Note: It wouldn't be unreasonable to make this an official dependency, but I didn't want to add it in case you have good reasons for not including it in the dependency list.